nest_local_flask/benchmark_plots.py

```diff
@@ -0,0 +1,234 @@
+import numpy as np
```

Comment:
Please add docstring at the top saying what this file does.

"""Makes Flask nest plots

More info More info More info More info More info More info More info More info More info More info More info More info More info More info More info More info More info More info More info More info More info More info More info More info More info More info More info More info More info More info More info More info More info More info 
"""